src/object/cli_obj.c

```diff
@@ -2691,8 +3273,6 @@ obj_comp_cb(tse_task_t *task, void *data)
 				obj_auxi->io_retry = 0;
 			}
 		}
-		if (!obj_auxi->spec_shard && task->dt_result == -DER_INPROGRESS)
-			obj_auxi->to_leader = 1;
```

Comment:
why remove above two lines?

Reply:
ah, it suppose to be in the following patches, I will add it back.